packages/react-core/src/components/Tabs/examples/TabsHorizontalOverflow.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Tabs, Tab, TabTitleText, Checkbox } from '@patternfly/react-core';
+
+export const TabsHorizontalOverflow: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number>(0);
+  const [showTabCount, setShowTabCount] = React.useState(false);
+
+  const handleTabClick = (_event: any, tabIndex: string | number) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  return (
+    <div>
+      <Tabs
+        activeKey={activeTabKey}
+        onSelect={handleTabClick}
+        aria-label="Tabs in the default overflow example"
```

Comment:
I think there are a couple a11y violations about duplicate accessible labels because you need to update this aria-label to be 'Tabs in the horizontal overflow example' and the first Tab's aria-label to be 'Horizontal overflow content users"